src/component/Display.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+
+export default function Display(props) {
+  return (
+    <div>
+      <input placeholder="enter the number" value={props.value} />
+    </div>
+  );
+}
```

Comment:
- `value` props should be called `"result"` instead according to requirement, [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png)]
- it should also be a string, re-inforce this with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)]

- this prop's default value should be `'0'`, confirm this with `defaultProps` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]

- Also, why should it be inside an `input`, it's a `DISPLAY` component, it should just display the value of this prop at the moment

Reply:
the last point will be updated in milestone 4 or 5 when I will use the event handles